README.md

```diff
@@ -1,20 +1,41 @@
 # Rails Dynamic Request Lab
 
+
 ## Objectives
 
-1. Draw a route with a dynamic variable
-2. Use a routes variable in params within the logic of an action
-3. Assign an instance variable in a controller action
-4. Use a controller instance variable to generate a dynamic ERB template.
-5. Build a classical show action/view
+* Draw a route with a dynamic variable
+
+* Use a routes variable in params within the logic of an action
+
+* Assign an instance variable in a controller action
+
+* Use a controller instance variable to generate a dynamic ERB template.
+
+* Build a classical show action/view
+
+
+## Instructions
+
+There are three RSpec/Capybara specs in this lesson, you can run them with the command ```rspec spec/features/student_spec.rb```, make sure they pass without breaking the other tests, to complete this section all specs should pass.
+
+In this lab you will work through the full MVC pattern, at a high level you will integrate the following scenario in your application:
+
+1. Your router will be able to process dynamic requests URLs such as: ```students/5``` where ```5``` is a student ID.
+
+2. That request will be routed through a new controller ```show``` action in your students' controller.
+
+3. The show method will parse the ```:id``` route variable and pass it into a query on the Student model and store it in an instance variable.
+
+4. Finally the student show template will render the values from the instance variable relayed from the show action.
```

Comment:
I try and keep the directions as more feature descriptions/user stories. So instead of this very sort of rails mechancis description, I would do something like "The user is able to go to `/students/2` to receive information on the student with the id of 2". IT's ok if these are short